mkosi/__init__.py

```diff
@@ -4836,6 +4885,9 @@ def run_verb(args: Args, images: Sequence[Config], *, resources: Path) -> None:
         page(text, args.pager)
         return
 
+    if args.verb == Verb.bump:
```

Comment:
Why move this down?

Reply:
I wanted to move it at least after https://github.com/systemd/mkosi/blob/6710479778d03eb05ebe8dfe972c53737fa34f75/mkosi/__init__.py#L4821 just to ensure no default configs are used, as it now depends on a config being present. And after `summary` seemed like a fine position but I am unsure if it actually should now also be after `ensure_root_is_mountpoint()`.